### PR TITLE
[enhancement] document startup flags

### PR DESCRIPTION
**Pull request by line-o**
refs https://github.com/eXist-db/exist/pull/4196

### CODE REVIEW

**Comment by joewiz** [comment]
@line-o Thanks! The CI failures indicate that `mvn validate` is raising schema errors.

**Comment by line-o** [comment]
Yes, apparently. docBook does require some training it seems.

**Comment by line-o** [comment]
OK, the machine has accepted my edits.

**Comment by joewiz** [comment]
@eXist-db/documenters Review away! But please do not merge this until https://github.com/eXist-db/documentation/issues/742 is closed.

**Review by dizzzz — APPROVED** [review]

File: src/main/xar-resources/data/advanced-installation/advanced-installation.xml [hunk 1]
```diff
@@ -185,4 +188,43 @@
             </varlistentry>
         </variablelist>
     </sect1>
+    <sect1 xml:id="cli-flags">
+        <title>Command line flags to set host, ports and configuration</title>
+        <para>There are several java tool options available to change the host, http port
```
Comment (line-o) [inline src/main/xar-resources/data/advanced-installation/advanced-installation.xml:193]: http ~~port~~ and

Comment (joewiz) [reply src/main/xar-resources/data/advanced-installation/advanced-installation.xml:193]: @line-o Am I understanding correctly from your review here that there are still planned changes on this PR? I'll hold off on merging until you confirm it's ready.

Comment (line-o) [reply src/main/xar-resources/data/advanced-installation/advanced-installation.xml:193]: I'll do that now. So that we can get this merged.

**Review by joewiz — APPROVED** [review]

**Pull request merged**